source/_integrations/oralb.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: OralB
+description: Get information about your Bluetooth OralB toothbrush
+ha_category:
+  - Sensor
+ha_bluetooth: true
+ha_release: 2022.11
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@bkbilly'
+ha_domain: oralb
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_supporting_domain: oralb
+ha_supporting_integration: OralB
+---
+
+{% include integrations/supported_brand.md %}
```

Comment:
This isn't a supported brand?

Reply:
Hi, I am not sure what you mean.
It's the 1st time I am creating a new integration on Home Assistant and I found this template from another integration and I wrote this.
Should I have written something else?

Reply:
Yeah, oralB seems like an integration on its own, and is not provided by something else. Have you checked the output? 

<https://deploy-preview-24560--home-assistant-docs.netlify.app/integrations/oralb/>

The template you have been using is used in case support for brand X is provided by integration Y, which is not the case right now.

Please take a look at other integration pages that use `{% include integrations/config_flow.md %}`.

Reply:
Thanks for your help.
I've just updated it with the config_flow template.